asymmetric.go

```diff
@@ -285,6 +285,9 @@ func (ctx rsaDecrypterSigner) signPayload(payload []byte, alg SignatureAlgorithm
 
 	switch alg {
 	case RS256, RS384, RS512:
+		// TODO: As of go1.20, the random parameter is legacy and ignored, and it
```

Comment:
File and indicate an issue for this TODO.

```suggestion
		// TODO(#<issue>): As of go1.20, the random parameter is legacy and ignored, and it
```